File: src/luogu/p1271/Main.cpp
// https://www.luogu.com.cn/problem/P1271
#include <iostream>
using namespace std;
int p[1000];
int main() {
  int n, m;
  cin >> n >> m;
  for (int i = 0; i < m; i++) {
    int tmp;
    cin >> tmp;
    p[tmp]++;
  }
  for (int i = 1; i <= n; i++) {
    if (p[i] == 0)
      continue;
    for (int j = 0; j < p[i]; j++) {
      cout << i << " ";
    }
  }
  cout << endl;
  return 0;
}

File: src/luogu/p1271/Main.py
# https://www.luogu.com.cn/problem/P1271
(n, m) = list(map(int, input().split()))
p = [0 for i in range(n + 1)]
t = list(map(int, input().split()))
for i in t:
    p[i] += 1
for i in range(len(p)):
    if p[i] == 0:
        continue
    for j in range(p[i]):
        print(i, end=' ')
print()
